src/interfaces/todo-i.ts

```diff
@@ -0,0 +1,70 @@
+import type { FilingTypes } from '@bcrs-shared-components/enums'
+import type { PaymentMethodE } from '~/enums/payment-method-e'
+import type { PaymentErrorI } from '~/interfaces/payment-error-i'
+import type { FilingSubTypeE } from '~/enums/filing-sub-type-e'
```

Comment:
fyi you can reference these from '#imports' instead of the specific file. Is nice to do it that way in case we ever move things around (i.e. into a common repo or different file)